packages/components/src/components/icon/library.system.ts

```diff
@@ -18,6 +18,8 @@ const icons = {
 		<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" fill='currentColor'><path d="M8.79496 15.875L4.62496 11.705L3.20496 13.115L8.79496 18.705L20.795 6.70498L19.385 5.29498L8.79496 15.875Z"/></svg>`,
 	'chevron-down': `
 		<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" fill='currentColor'><path d="M16.59 8.29498L12 12.875L7.41 8.29498L6 9.70498L12 15.705L18 9.70498L16.59 8.29498Z"/></svg>`,
+	'chevron_right': `
```

Comment:
```suggestion
	'chevron-right': `
```

We have `chevron-down`

Reply:
I did – as well for `more-vert` and `chevron-right`.

Reply:
I updated the system icons